plugins/database/influxdb/connection_producer.go

```diff
@@ -257,7 +257,7 @@ func isUserAdmin(cli influx.Client, user string) (bool, error) {
 	for _, res := range response.Results {
 		for _, serie := range res.Series {
 			for _, val := range serie.Values {
-				if val[0].(string) == user && val[1].(bool) == true {
+				if val[0].(string) == user && val[1].(bool) {
```

Comment:
Obviously not the intention of your PR, but couldn't the type assertions here panic?

Reply:
Eh, it's probably fine. As in many DB drivers, the columns and values are split across separate fields. Eeach entry of `Row.Values` is `[]interface{}` thus the type assertions. I have not used influxdb but it looks like a `SHOW USERS` query would output something like:

```
SHOW USERS
user    admin
----    -----
admin   true
```

Feel free to ignore me :)